### ISSUE TITLE
A couple of feature requests

### ISSUE THREAD

**Issue by badlydrawnrob** [trobonox/kanri#619]
First quick impressions after using the app:

1. Current day/date visible on calendar
2. Quick Github style tasks (understand this could be a significant UI change if integrating with current tasks)
3. Better "overflow" solution to too many tasks in view (currently scrolls)

<img width="220" alt="Screenshot 2024-08-14 at 18 38 43" src="https://github.com/user-attachments/assets/b73188bc-c5a4-409e-8745-1d2816215d6f">

**Comment by trobonox** [comment]
Thanks for your feedback! Allowing some kind of tip is definitely something I want to do, but there's some more setup I need to do in order to make it work.

For the little feature requests, these are some good insights. I will take a look at each one and see what I can do (especially the markdown suggestions might need some extra work and consideration).

Will keep you updated with the progress. Since some features might take longer or need extra discussion, I might extract those into separate issues.

**Comment by badlydrawnrob** [comment]
Sure. GoCardless is one such option if you want to encourage one-off mobile payments. I agree adding features without thought to how it affects the whole is a bad idea.

I'm finding a few bugs — would you prefer I log them one by one as issues?

**Comment by trobonox** [comment]
> I'm finding a few bugs — would you prefer I log them one by one as issues?

Yes, I would appreciate it if you could create the bug reports as individual issues.

**Comment by badlydrawnrob** [comment]
I updated my post with a couple of suggestions ;)

**Comment by trobonox** [comment]
Added a little outline which always highlights the current day in the due date picker. This change will be out with the next release.

The rest of the ideas will need some extra discussion since I'm not sure about them yet.
